fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/LoanAccountDomainServiceJpa.java

```diff
@@ -437,6 +438,8 @@ public void reverseTransfer(final LoanTransaction loanTransaction) {
     @Override
     public void recalculateAccruals(Loan loan) {
         LocalDate accruedTill = loan.getAccruedTill();
+        boolean isOrganisationDateEnabled = this.configurationDomainService.isOrganisationstartDateEnable();
+        Date organisationStartDate = this.configurationDomainService.retriveOrganisationStartDate();
```

Comment:
Do this only when Start Date is enabled.... one DB query can be avoided....
Also it is better to keep API spellings clean